PhotoMap/Features/Timeline/ViewModels/TimelineViewModel.swift

```diff
@@ -88,8 +108,25 @@ class TimelineViewModel: TimelineViewModelType {
                 groupedMarkers[marker.date.monthAndYear] = [marker]
             }
         }
-        self.markers = groupedMarkers
-        self.headerTitles = [String](groupedMarkers.keys).sorted { $0.toMonthAndYearDate > $1.toMonthAndYearDate }
+        let titlies = [String](groupedMarkers.keys).sorted { $0.toMonthAndYearDate > $1.toMonthAndYearDate }
+        return (groupedMarkers, titlies)
+    }
+    
+    private func searchMarkersWith(hashtag: String) {
+        let allMarkers = self.markers.values.flatMap { $0 }
+        var filteredMarkers = [Marker]()
+        
+        markersLoop: for marker in allMarkers {
+            for tag in marker.hashtags {
```

Comment:
Combine `for` and `if`:
```suggestion
            for tag in marker.hashtags where tag.lowercased().contains(hashtag.lowercased()) {
```